miraclelinux/meta-emlinux
# Add compiler settings for security

# Purpose of pull request

This pr adds compiler options to sdk setup scripts.

# Test
## How to test

Build sdk as follows.

```
$ bitbake emlinux-image-base -c populate_sdk
```

After building,  install sdk and run setup script.  
And then check some environment variables such as CC , CPP, CFLAGS , ...

## Test result

```
$ echo $CC
/home/okada/qoriq-sdk-chk/emlinux-image-base-sdk-emlinux-bookworm-ls1046ardb/usr/bin/aarch64-linux-gnu-gcc -fstack-protector-strong -O2 -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security
```
